### DISCUSSION TITLE
I don't know who's right of the Raylib Software.

### DISCUSSION THREAD

**Discussion by ian1ilych** [raysan5/raylib#2545] [Polls]
I want to use Raylib on my PC, because none of these exe files worked.
And my Avast Software detects it as an unverified file.

I opened https://virustotal.com but 1 antivirus detects it as malware.

I don't know who's right of malware of Raylib, it was have a choice to know about.
It could be a false positive.

Let me know.

**Comment by orcmid** [comment]
It makes no sense to ask for a poll here.  

Please provide complete information.  What is your computer.  Where are you obtaining raylib?  What are you scanning - the download before installing or after installing?

With regard to the malware report, what of raylib is being identified as malware?

And if only one antivirus claims it and others do not, isn't that a poll that you already distrust?

**Reply by ian1ilych** [reply]
HP EliteBook Folio 9480m

**Reply by ian1ilych** [reply]
From this source.

**Reply by ian1ilych** [reply]
Antivirus (Website): VirusTotal

**Comment by raysan5** [comment]
raylib is open source software and with a relatively small codebase. As far as I know, there is no malicious code in there but you can review the code yourself. Some antivirus could detect some C code patterns as false positives.

**Reply by ian1ilych** [reply]
Thanks, now i can use it..
I think it will be easy for me to make a game.
i think..

**Comment by ian1ilych** [comment]
It is to make sure it was a false positive.El 23 jun. 2022 8:18 p. m., "Dennis E. Hamilton" ***@***.***> escribió:
It makes no sense to ask for a poll here.
Please provide complete information.  What is your computer.  Where are you obtaining raylib?  What are you scanning - the download before installing or after installing?
With regard to the malware report, what of raylib is being identified as malware?
And if only one antivirus claims it and others do not, isn't that a poll that you already distrust?

**Reply by ian1ilych** [reply]
It happened when I reply it to my email.

**Comment by CrezyDud** [comment]
dont be overly scared with everything,
just look at the source, maybe some games made with raylib, see how they were made
and maybe look at raylib itself,
then make some stuff

**Reply by ian1ilych** [reply]
I guess you're right, but i'm gonna find school for programming, since it's my first time on programming. i found the nearest one, but it takes a long ride😭.
